ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx

```diff
@@ -30,11 +31,12 @@ export class Dropdown {
   render() {
     return (
       <div class="ontotext-dropdown">
-        <button class={`ontotext-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
-                onClick={() => this.toggleComponent()}>
-          {this.translationService.translate(this.nameLabelKey)}
-        </button>
-
+        <div class="ontotext-dropdown-button-wrapper" onClick={() => this.toggleComponent()}>
```

Comment:
Using non actionable elements like a `div` with a clickhandler is bad from UX point of view. Better, move the icon inside the button and return the clickhandler on it.

Reply:
I had problem with styling when icon is inside the button but I will try again.